### PR TITLE
Document bugs found in Hostboot

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

File: devnotes/hostbug.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+uint8_t getMCPortNum(uint8_t i_portID)
+{
+    uint8_t l_mcPos = getMCPosition(i_portID);
+    return (i_portID - (2 * l_mcPos));
+}
+```
+
+### No information about changed SCOM addresses
+
+SCOM addresses has changed between DD1 and DD2 nit there is no information about
```
Comment (macpijan) [inline devnotes/hostbug.md:80]: but?

File: devnotes/hostbug.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+bool PlatPmPPB::isPstateModeEnabled()
+{
+    return((iv_attrs.attr_pstate_mode == fapi2::ENUM_ATTR_SYSTEM_PSTATES_MODE_OFF) ?
+            true : false);
+} //end of isPstateModeEnabled
+```
+
+### Different names for the same attribute
+
+Są jakby aliasy pod innymi nazwami i z innym sposobem dostępu do nich. Na przykładzie https://git.raptorcs.com/git/talos-hostboot/tree/src/usr/targeting/common/xmltohb/attribute_types.xml#n3583 mamy MIN_FREQ_MHZ, które jest nazwą hostbootowego atrybutu, i ATTR_FREQ_CORE_FLOOR_MHZ, które jest tym samym w FAPI/HWPF. Pierwszy z nich jest używany chyba wszędzie poza src/import/*, drugi tylko tam i w kodzie powstającym z src/usr/targeting, który mapuje między jednymi a drugimi (w obj/genfiles dużo plików się pojawia po kompilacji).
```
Comment (macpijan) [inline devnotes/hostbug.md:97]: english, please

File: devnotes/hostbug.md (hunk 1 above)
Comment (IgorBagnucki) [reply devnotes/hostbug.md]: Fixed

File: devnotes/hostbug.md (hunk 2 above)
Comment (IgorBagnucki) [reply devnotes/hostbug.md]: Left it by mistake. Fixed

**Pull request merged**